pom.xml

```diff
@@ -413,6 +461,53 @@
           <artifactId>maven-failsafe-plugin</artifactId>
           <version>3.0.0-M3</version>
         </plugin>
+        <!-- Required to build in eclipse = ignoring not supported plugins -->
+		<plugin>
+			<groupId>org.eclipse.m2e</groupId>
+			<artifactId>lifecycle-mapping</artifactId>
+			<version>1.0.0</version>
+			<configuration>
+		     <lifecycleMappingMetadata>
+			  <pluginExecutions>
+	           <pluginExecution>
+			    <pluginExecutionFilter>
+				 <groupId>au.com.acegi</groupId>
+				 <artifactId>xml-format-maven-plugin</artifactId>
+				 <versionRange>[3.1,)</versionRange>
+				 <goals><goal>xml-check</goal></goals>
+			   </pluginExecutionFilter>
+			   <action><execute /></action>
+			 </pluginExecution>
+	         <pluginExecution>
```

Comment:
There's a good chance CI is going to fail on this due to formatting. From command line you'll want to run `mvn xml-format:xml-format fmt:format` and repush.